javalib/src/main/scala/java/util/SplittableRandom.scala

```diff
@@ -106,27 +108,8 @@ final class SplittableRandom private (private var seed: Long, gamma: Long) {
     seed
   }
 
-  def nextInt(): Int = mix32(nextSeed())
-
-  //def nextInt(bound: Int): Int
-
-  //def nextInt(origin: Int, bound: Int): Int
+  override def nextInt(): Int = mix32(nextSeed())
 
   def nextLong(): Long = mix64(nextSeed())
 
```

Comment:
Add override for `nextBytes` for interface compatiblity? (Seems fine to delegate to super, IDK how we usually handle this, alternatively just comment)?

https://docs.oracle.com/en/java/javase/23/docs/api/java.base/java/util/SplittableRandom.html#nextBytes(byte%5B%5D)

Reply:
Since `SplittableRandom` is a `final class`, it doesn't make a difference. If it were open, then we could have classes that inherit from it, and they would need to have the right methods in their *class* hierarchy so that they take priority over default methods in interfaces. But as a `final class` none of this is observable.